datasets/woningbouwplannen/Woningbouwplannen en strategische ruimtes.json

```diff
@@ -0,0 +1,338 @@
+          },
+          "ligtinGbdBrtIdentificatie": {
+            "type": "string",
+            "provenance": "ligtin_gbd_brt_identificatie",
+            "description": "Unieke identificatie van de buurt."
+          },
+          "ligtinGbdBrtVolgnummer": {
+            "type": "string",
+            "provenance": "ligtin_gbd_brt_volgnummer",
+            "description": "Uniek volgnummer van de toestand van de buurt"
+          },
+          "aantalWoningen": {
+            "type": "string",
+            "provenance": "aantal_woningen",
+            "description": "Aantal woningen in het bouwblok met de volgende gebruiksdoelen: 1- Woonfunctie, 2- In combinatie met een woonfunctie. Status: 1- Verblijfsobject gevormd, 2- Niet gerealiseerd verblijfsobject, 3- Verblijfsobject in gebruik (niet ingemeten), 4- Verblijfsobject in gebruik, 5- Verbouwing verblijfsobject"
+          },
+          "aantalVerblijfsobjecten": {
+            "type": "string",
+            "provenance": "aantal_verblijfsobjecten",
+            "description": "Alle verblijfsobjecten in het bouwblok zonder filtering op status en gebruiksdoel."
+          },
+          "wijzigingsdatumDp": {
+              "type": "string",
+              "provenance": "wijzigingsdatum_dp",
+              "format": "date",
+              "description": "Datum waarop het object is gewijzigd"
+          },
+          "verwijderdDp": {
+            "type": "boolean",
+            "provenance": "verwijderd_dp",
+            "description": "indicatie of het object verwijderd is bij de bronhouder"
+        }
+      }
+    }
+  }, 
+    {
+        "id": "strategischeruimtes",
+        "type": "table",
+        "provenance": "wbw_strategischeruimte",
+        "title": "Strategischeruimtes",
+        "description": "De aantallen vormen de planvoorraad. Dit zijn niet de aantallen die definitief worden gerealiseerd. Ervaring leert dat een deel van de planvoorraad wordt opgeschoven. Niet alle woningbouw initiatieven doorlopen de verschillende plaberumfasen. Met name kleinere particuliere projecten worden in de regel pas toegevoegd aan de monitor zodra er een intentieovereenkomst of afsprakenbrief is getekend.",
+        "schema": {
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "type": "id",
+          "additionalProperties": false,
+          "required": ["schema", "id"],
+          "mainGeometry": "geometrie",
+          "display": "id",
+          "properties": {
+            "id": {
+              "type": "integer",
+              "description": "Unieke id van het object",
+              "provenance": "object_id"
+            },
+            "schema": {
+              "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+            },
+            "projectnaam": {
+              "type": "string",
+              "description": "Naam van het project"
+            },
+            "cluster_id": {
```

Comment:
Omdat de naam identiek is aan de provenance, is de provenance niet nodig. De naam moet wel sowieso in camelCase, dus in dit geval "clusterId"